Include/internal/pycore_object.h

```diff
@@ -10,6 +10,10 @@ extern "C" {
 
 #include "pycore_pystate.h"   /* _PyRuntime */
 
+/* Private API for the LOAD_METHOD opcode. */
+PyAPI_FUNC(int)
+_PyObject_GetMethod(PyObject *obj, PyObject *name, PyObject **method);
```

Comment:
Other API definitions in this file are single line format.

Reply:
BTW, I'm not sure we need `PyAPI_FUNC` here.
It is used to expose symbol.  But this API is really private.  This is used only in python executable or python dll.
Even extension modules in stdlib must not use it.

Reply:
> BTW, I'm not sure we need `PyAPI_FUNC` here.

It seems that most functions in the internal API do use `PyAPI_FUNC`. So it seems better to keep it. If you insist, I will remove the `PyAPI_FUNC`.